### PR TITLE
Log sink for events

### PR DESCRIPTION
**Pull request by mwielgus**
cc: @piosz

### CODE REVIEW

**Comment by huangyuqi** [comment]
Hi, @mwielgus  I inadvertently found this `heapster-scalability` branch ,  is there a reconstruction plan of heapster in community? would you please share me some details about this. Thanks so much.

**Comment by mwielgus** [comment]
Yes, we are significantly refactoring Heapster. It was announced in #684.
More details on the future of Heapster can be also found here: #769

File: events/sinks/log/log_sink.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	return "LogSink"
+}
+
+func (this *LogSink) Stop() {
+	// Do nothing.
+}
+
+func batchToString(batch *core.EventBatch) string {
+	var buffer bytes.Buffer
+	buffer.WriteString(fmt.Sprintf("EventBatch     Timestamp: %s\n", batch.Timestamp))
```
Comment (gmarek) [inline events/sinks/log/log_sink.go]: An idea: maybe use text/tabwriter to this?

**Comment by gmarek** [comment]
One suggestion, but otherwise LGTM. Ping me if you decide to use tabwriter.

File: events/sinks/log/log_sink.go (hunk 1 above)
Comment (mwielgus) [inline events/sinks/log/log_sink.go]: This is only for debugging. No need for aligning.

**Comment by huangyuqi** [comment]
Thanks, @mwielgus , shall I join some Heapster refactoring contribution work? Or assign me some task about this? ^_^

**Comment by mwielgus** [comment]
Sure, any help will be more than welcome. I will try to produce a list of pending tasks Today but if I don't (Christmas is coming) you can start with porting Kafka and Riemann sinks to the new api.

**Comment by huangyuqi** [comment]
Thanks so much for letting me join. ^_^
May you have the best Christmas ever. @mwielgus 
I will invest in the porting work today and wait for your pending tasks list.

**Pull request merged**